### PR TITLE
Adding C# FaceLocator template using Computer Vision Api

### PR DESCRIPTION
**Pull request by soninaren**

### CODE REVIEW

**Comment by soninaren** [comment]
@fabiocav  can you take a look at this PR?

**Comment by soninaren** [comment]
Addresses #217

**Comment by mathewc** [comment]
Want to use this PR as the forcing function for a new "Samples" category? Then we add this new sample to that category, as well as the following existing templates:

GitHubCommenter-CSharp
HttpGET(CRUD)-CSharp
HttpGET(CRUD)-NodeJS
HttpPOST(CRUD)-CSharp
HttpPOST(CRUD)-NodeJS
ImageResizer-CSharp

File: Templates/FaceLocator-CSharp/run.csx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+{
+    [JsonProperty("faces")]
+    public List<Face> faces { get; set; }
+}
+
+public class Face
+{
+
+    [JsonProperty("age")]
+    public int age { get; set; }
```
Comment (fabiocav) [inline Templates/FaceLocator-CSharp/run.csx]: nit: Since we're using the JsonProperty attribute, use the usual casing convention for properties (Pascal)

Comment (fabiocav) [inline Templates/FaceLocator-CSharp/run.csx]: Removing the JSON attributes would clear up a lot of the code in the sample, though. As the types are defined today, they're not really required (except for maybe a JsonIgnore)

File: Templates/FaceLocator-CSharp/run.csx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+#r "Newtonsoft.Json"
+
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using Newtonsoft.Json;
+using Microsoft.WindowsAzure.Storage.Table;
+using System.IO;
+
+public static async Task Run(Stream image, string name, ICollector<FaceRectangle> outTable, TraceWriter log)
```
Comment (fabiocav) [inline Templates/FaceLocator-CSharp/run.csx]: Change this to use an `IAsyncCollector<FaceRectangle>` and change the code to use `AddAsync`

**Comment by soninaren** [comment]
updated as per comments

File: Templates/FaceLocator-CSharp/run.csx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    public List<Face> Faces { get; set; }
+}
+
+public class Face
+{
+    public int Age { get; set; }
+
+    public string Gender { get; set; }
+
+    public FaceRectangle faceRectangle { get; set; }
```
Comment (mathewc) [inline Templates/FaceLocator-CSharp/run.csx]: fix casing

Comment (soninaren) [inline Templates/FaceLocator-CSharp/run.csx]: fixed in #222

**Pull request merged**